### ISSUE TITLE
Recalculate auto armor module

### ISSUE THREAD

**Issue by base64-sys** [MeteorDevelopment/meteor-client#4933]
### Describe the feature

Auto armor basically sorts through armor and selects ( + equips ) the ones with the highest protection enchantment. Well I'm suggesting that instead of focusing only on protection, it focuses on more details. For example, right now, if I use auto armor with a diamond chestplate with protection 3 and a netherite chestplate with protection 2, i would like the netherite one better.

Basically i want it to select the best one overall, not only protection enchantment-wise

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)
